src/store/reducers.js

```diff
@@ -0,0 +1,53 @@
+import {
+  SET_USER,
+  UPDATE_USER,
+  REMOVE_USER,
+  ADD_TASK,
+  GET_TASKS,
+  GET_INFO
+} from './actions';
+
+
+export const user = (state = false, { type, data }) => {
+  switch (type) {
+    case UPDATE_USER:
+    case SET_USER: {
+      return data;
+    }
+
+    case REMOVE_USER:
+      return null;
+  }
+
+  return state;
+};
+
+export const taskList = (state = [], { type, data }) => {
+  switch (type) {
+    case ADD_TASK: {
+      return state.map((day) => {
+        if (day === data.day) {
+          return [...day, data];
```

Comment:
when this code is executed? I wasn't able to step into this line while debugging